.github/workflows/release.yml

```diff
@@ -117,30 +148,40 @@ jobs:
           TAG: ${{ github.ref }}
         run: echo "version=${TAG:11}" >> $GITHUB_OUTPUT
       - name: Fetch Linux artifact
-        uses: actions/download-artifact@v2
+        uses: actions/download-artifact@v4
```

Comment:
Let's do this separately! I am happy to take care of that, though it will be this evening or later this week; and you can merge those changes in or rebase on top of them once we have!

Reply:
Thanks, I understand. In this PR, I revert the version change of the download-artifact and create a separate PR to address it.